### ISSUE TITLE
Javascript exception

### ISSUE THREAD

**Issue by RvMouche** [danielbuechele/goofy#332]
Just got this exception, the app kept working and I couldn't find it in the inspector nor reproduce
![screen shot 2017-03-02 at 18 29 32](https://cloud.githubusercontent.com/assets/2125224/23519203/64fb1d9c-ff76-11e6-937e-cee24094d083.png)
 it.

**Comment by juandaco** [comment]
I'm having the same issue in MacOS Sierra.

**Comment by RvMouche** [comment]
I managed to reproduce the exception @danielbuechele. It happens on every check for update.. This one was just caused by the auto-check I assume.

**Comment by danielbuechele** [comment]
So this happens everytime you call "Check for Update" from the menu?

**Comment by RvMouche** [comment]
Yes, and one some "random" moments that are presumably the automatic checks from Sparkle

**Comment by mrleblanc101** [comment]
I had this problem. Seem to have something to do with the beta version ! Remove it and download the official version from the website and it should fix this.

**Comment by joshrickert** [comment]
I was also getting this on startup. Reinstalling 3.0.3 seems to have fixed it.

**Comment by joshrickert** [comment]
Just got the same error again launching 3.0.3. But when closing and relaunching the app, it didn't appear again.

**Comment by 030media** [comment]
I have the exact same issue with version 3.4.6 and earlier. It just happened at one point and never disappeared no matter how many reinstalls and updates I made.
